services/diversity/src/services/richness.js

```diff
@@ -110,6 +110,48 @@ module.exports = () => {
           return data;
       }
     },
+
+    /**
+     * Get values for richness species gaps in the given area
+     *
+     * @param {String | Number} areaType area type
+     * @param {String} areaId area id
+     *
+     * @returns {Object} Values of richness species gaps
+     */
+    // eslint-disable-next-line no-unused-vars
```

Comment:
don't disable eslint, just delete the parameters but leave them documented so we can add them when implement the real function

Reply:
Fixed